### ISSUE TITLE
oops... something went terribly wrong

### ISSUE THREAD

**Issue by Rubilu** [Open-Systems-Pharmacology/Forum#314]
Hello everyone,

I've updated to 7.4 and in my population approach, the software changes the observed data - but only when I choose the parent drug. The metabolite data is correct.
For example: The cmax of the observed data is 65,26 µg/L with a standard deviation of 4,46. The simulation changes it into a cmax of 61,96 µg/L with a SD of 4,2. The metabolite cmax is identical in the simulation. When I take the same values for my parent drug but characterize them as values for the metabolite, the values are the same in the population approach. So, it seems to be a problem with the linking between parent drug and observed data.
I don't know how to fix this bug and hope that someone can help me.

Kind regards

**Comment by StephanSchaller** [comment]
I am not fully certain if I understand your issue correctly, but could this be caused by the correction factor accounting for halogens in your parent drug?
How did you load the observed data? As molar or mass dimension?

**Comment by Rubilu** [comment]
There is no halogen in my parent drug. I've loaded the observed data as mass dimension. 
I try to specify my issue: The software reduces the observed data values of the parent drug but not the observed data values of the metabolite. In my opinion there should not be a change of the observed data at all.

**Comment by StephanSchaller** [comment]
> In my opinion there should not be a change of the observed data at all.

No, there shouldn't. This issue will need to be transferred to the PK-Sim bug-reporting repository. Do you have the project file that can be attached here to reproduce the issue (or is it confidential, then it would be great if you could list the steps you took to reproduce the issue)?
